projects/storefrontlib/cms-structure/services/cms-features.service.ts

```diff
@@ -169,9 +174,21 @@ export class CmsFeaturesService {
 
     // extract cms components configuration from feature config
     for (const componentType of featureConfig.cmsComponents ?? []) {
+      /**
+       * update same cms component mapping's configuration found
+       * across all feature instance if the configuration exist
+       * in order to use the latest configuration
+       **/
+      if (resolvedConfiguration.cmsComponents?.[componentType]) {
```

Comment:
Can we come up with a test which can test this functionality?
Ideally, we should cover it with a unit test.
An e2e is welcomed as well.